src/api/profile.ts

```diff
@@ -48,3 +48,17 @@ export const makeUpdateUserGridLocation = ({
       firestore.doc(doc).set(newData);
     });
 };
+
+export interface makeUpdateUserOnlineStatusProps {
+  userUid: string;
+}
+
+export const makeUpdateUserOnlineStatus = ({
+  userUid,
+}: makeUpdateUserOnlineStatusProps) => (status: string) => {
+  const newData = {
+    "data.status": status,
```

Comment:
I think we should store it on the top level.
```suggestion
    "status": status,
```